languages/anaconda3/template/algorithmia.conf

```diff
@@ -1,5 +1,5 @@
 {
     "username": "__USER__",
     "algoname": "__ALGO__",
-    "language": "anaconda3x"
+    "language": "conda3x"
```

Comment:
hmm this is a difficult one, we should call it 4x, but changing this would require also modifying additional locations in `algorithmiahq/common`, etc which would require multiple redeploys... Lets keep this as `anaconda3x` for now, and continue testing in test + caching logic built out in the sister PR. If everything works then I would argue for changing the language and modifying the upstream systems.

Reply:
Agree, done.